data/campaigns/Eastern_Invasion/scenarios/12_Evacuation.cfg

```diff
@@ -53,9 +53,9 @@
         {FLAG_VARIANT6 ragged}
     [/side]
 
-    {LIMIT_CONTEMPORANEOUS_RECRUITS 2 "Direwolf Rider" 3}
+    {LIMIT_CONTEMPORANEOUS_RECRUITS 2 "Direwolf Rider" {ON_DIFFICULTY 1 2 3}}
 
-    {STARTING_VILLAGES 2 8}
+    {STARTING_VILLAGES 2 {ON_DIFFICULTY 4 6 8}}
```

Comment:
Same thing as the other comment, I don't think varying the villages so much based on difficulty is good. If you want to keep this, I think it should be 6 7 8.

Reply:
ok, changed in 33a48a5